payments/core.py

```diff
@@ -32,8 +32,11 @@ def get_base_url():
     """
     protocol = 'https' if PAYMENT_USES_SSL else 'http'
     if not PAYMENT_HOST:
-        current_site = Site.objects.get_current()
-        domain = current_site.domain
+        try:
+            current_site = Site.objects.get_current(request)
+            domain = current_site.domain
+        except Site.DoesNotExist:
+            domain = request.get_host()
```

Comment:
This will raise an exception when `request` is `None`.

Reply:
you're right. I fix it in next commit.